snippets/rust.snippets

```diff
@@ -147,7 +147,11 @@ snippet loop "loop {}" b
 		${0:${VISUAL}}
 	}
 snippet wh "while loop"
-	while ${1:condition} {
+	while ${1:true} {
```

Comment:
This is not the idiomatic way to make a loop with no conditional in rust. could you revert this one please? Thanks

Reply:
It is fixed in the latest commit.